src/ape_plugins/_cli.py

```diff
@@ -22,38 +22,89 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _remove_prefix_add_version(set_convert, _prefix, version):
+    output = set()
+    text = None
+    if set_convert:
+        for i in set_convert:
+            if i.startswith(_prefix):
+                text = i[len(_prefix) :]
+            if version:
+                output.add(f"{text}    ({version})")
+            else:
+                output.add(text)
+        return output
+    else:
+        return None
+
+
+@cli.command(name="list", short_help="Display all plugins installed and available (including Core)")
```

Comment:
```suggestion
@cli.command(name="list", short_help="Display plugins")
```

This `short_help` is a little misleading.

The core and uninstalled plugins only get displayed via special option `-a`.

Reply:
Copy that thank you